README.rst

```diff
@@ -112,42 +116,57 @@ The ``aws-encryption-cli`` also includes the encryption context in the metadata
+If you provide an encryption context in a decrypt command, the ``aws-encryption-cli`` 
+verifies that the message being decrypted was encrypted with an encryption context that 
+meets the following requirements.
 
-If you provide ``key=value`` pairs, the encryption context in the encrypted message must contain matching pairs.
+If you provide ``key=value`` pairs, the encryption context in the encrypted message must 
+contain matching pairs.
 
 .. code-block:: sh
 
    --encryption-context required_key=required_value classification=secret
 
-If you provide only ``key`` elements, the encryption context in the encrypted message must contain those keys, regardless of the values. You can mix ``key`` and ``key=value`` elements in the same parameter value.
+If you provide only ``key`` elements, the encryption context in the encrypted message 
+must contain those keys, regardless of the values. You can mix ``key`` and ``key=value`` 
+elements in the same parameter value.
 
 .. code-block:: sh
 
    --encryption-context required_key classification=secret
 
-If the encryption context check fails, the output metadata includes information about the failure. For more information, , see `Encryption Context Failures`_.
+If the encryption context check fails, the output metadata includes additional 
+information about the failure. For more information, , see 
+`Encryption Context Failures`_.
    
 .. warning::
 
-   The ``aws-encryption-cli`` deletes any existing output files before checking the encryption context. If the encrypted message does not satisfy the encryption context requirements, the ``aws-encryption-cli`` the decrypt operation stops, but the deleted output file is not restored.
+   The ``aws-encryption-cli`` deletes any existing output files before checking 
+   the encryption context. If the encrypted message does not satisfy the encryption 
+   context requirements, the decrypt operation stops, but the deleted output file 
+   is not restored.
 
 
 Output Metadata
 ---------------
-In addition to the primary output of the operation, the commands generate useful metadata about the encrypt and decrypt operation. This metadata includes information about the operation as well as the complete header of the encrypted message.
+In addition to the primary output of the operation, the ``aws-encryption-cli`` generates
+useful metadata about the encrypt and decrypt operation. This metadata includes 
+information about the operation as well as the complete header of the encrypted message.
 
-The ``aws-encryption-cli`` writes the metadata to a text file that you specify. The metadata consists of a single line of formatted JSON for each cryptographic operation. When a command performs multiple operations, the ``aws-encryption-cli`` writes a separate line of JSON for each operation.
+The ``aws-encryption-cli`` writes the metadata to a text file that you specify. The 
```

Comment:
I still think we should take out the "text" differentiator.

Reply:
already changed in master